app/main/forms/work_order_forms.py

```diff
@@ -16,3 +17,19 @@ def __init__(self, brief_id, data_api_client, *args, **kwargs):
 
         responses = data_api_client.find_brief_responses(brief_id)
         self.seller.choices = [(br['supplierCode'], br['supplierName']) for br in responses['briefResponses']]
+
+
+class WorkOrderQuestionForm(DmForm):
+    def __init__(self, slug, *args, **kwargs):
+        super(WorkOrderQuestionForm, self).__init__(*args, **kwargs)
+        self.heading = questions[slug]['heading']
```

Comment:
We'll probably end up needing to make `questions` a parameter when we starting having different sets of questions, but I don't think that's a serious issue now.  (I'm just thinking aloud about how future-proof this design is.)